mainFunction.py

```diff
@@ -0,0 +1,24 @@
+#functions
```

Comment:
Don't you think it would be better to have just one import from an intermediate file like `import type_rules.py` and it contains all the rules files? 

Also moving all the rule files to a new folder to not mix every single file in the same directory like the main file and other purpose files?

Reply:
Boa ideia!